src/types/identity.ts

```diff
@@ -23,6 +25,10 @@ export type IdentityResponse = {
   addresses?: Address[]
   /** List of associated personal relationships */
   relations?: IdentityRelation[]
+  /** Date of the first connection */
+  createdAt: Date
+  /** Last connection sync date with the institution. */
```

Comment:
these comments don't seem to be accurate. We are referring to `IdentityResponse` here, not to "the connection"

Reply:
Thanks! You are right!